client/site-profiler/components/hosting-information/index.tsx

```diff
@@ -14,10 +14,15 @@ interface Props {
 export default function HostingInformation( props: Props ) {
 	const { dns = [], urlData, hostingProvider } = props;
 	const aRecordIps = dns.filter( ( x ) => x.type === 'A' && x.ip );
-	const supportUrl =
-		hostingProvider?.slug === 'automattic'
-			? localizeUrl( 'https://wordpress.com/help/contact' )
-			: hostingProvider?.support_url;
+	let supportUrl = null;
+
+	if ( hostingProvider?.slug === 'automattic' ) {
+		supportUrl = localizeUrl( 'https://wordpress.com/help/contact' );
+	} else if ( urlData?.platform_data?.support_url ) {
+		supportUrl = urlData.platform_data.support_url;
+	} else if ( hostingProvider?.support_url && ! hostingProvider?.is_cdn ) {
```

Comment:
I think no matter if it's CDN, the rest of the cases can apply to `hostingProvider?.support_url;` 

What do you think?

Reply:
Of course, not needed anymore.